src/services/exercise/handlers.ts

```diff
@@ -0,0 +1,131 @@
+import { responseMessages, type ReqHandler } from "@/lib/utils";
+import { exerciseSchemas } from "./schemas";
+import { prisma } from "@/lib/prisma";
+import { PrismaClientKnownRequestError } from "@prisma/client/runtime/library";
+
+const postExercise: ReqHandler = async (req, res, next) => {
+  try {
+    const inputBody = exerciseSchemas.postBody.safeParse(req.body);
+    if (!inputBody.success) {
+      return res.status(400).json({
+        message: responseMessages.error.reqBody,
+      });
+    }
+
+    const insertResult = await prisma.exercise.create({
+      data: inputBody.data,
+    });
+
+    const safeInsertResult = exerciseSchemas.base.parse(insertResult);
```

Comment:
no need to parse if you only need one property

Reply:
noted, I'll remove it